cloud/azure/bin/lib/azure_log.sh

```diff
@@ -75,6 +75,23 @@ function azure_log::log_file_exists() {
     | grep -q true
 }
 
+#######################################
+# Check role assignments and grants them if they don't exist
+# Globals read:
+#   AZURE_RESOURCE_GROUP
+#   AZURE_SUBSCRIPTION
+#   AZURE_LOG_STORAGE_ACCOUNT_NAME
+#######################################
+function azure_log::grant_log_role_assignments() {
```

Comment:
suggestion (blocking): recommend renaming to `ensure_log_role_assignments` since it does the check and bails if they already have it